assets/src/js/header.js

```diff
@@ -212,6 +212,17 @@ export const setupHeader = () => {
 
   const toggleElements = [...document.querySelectorAll(toggleElementClasses.join(','))];
 
+  /*
```

Comment:
@Osong-Michael I've pushed this proposed code if we think that is not necessary to set the `active` class in cases where there is an `#` as a part of the `pathname`. Take in mind that we can remove this commit if you disagree.  

What do you think? 
Also c.c. @comzeradd